src/canopy_cli/data_loader/errors.py

```diff
@@ -0,0 +1,37 @@
+import typing as t
+from click._compat import get_text_stderr
+from click import echo
+
+
+class IDsNotUniqueError(ValueError):
+    pass
+
+
+class DocumentsValidationError(ValueError):
+    pass
+
+
+class DataLoaderException(Exception):
+    """An exception that Click can handle and show to the user."""
+
+    #: The exit code for this exception.
+    exit_code = 1
+
+    def __init__(self, file_name: str, row_id: str, err: str) -> None:
+        message = f"""
+        [ERROR] {file_name} - {row_id} - {err}
```

Comment:
```suggestion
        {file_name}, line {row_id} - {err}
```

No need for the `Error` - the CLI adds this prefix anyway